edit-post/README.md

```diff
@@ -53,6 +54,22 @@ Title displayed at the top of the sidebar.
 - Type: `String`
 - Required: Yes
 
+##### isPinnable
+
+Whether to allow to pin sidebar to toolbar.
+
+- Type: `Boolean`
+- Required: No
+- Default: `true`
+
+##### icon
```

Comment:
Should this be a property of `registerPlugin` rather than a prop of `PluginSidebar` ?

Reply:
As I had a similar dilemma. I will explore how much work is required to move it to `registerPlugin`.

Reply:
I mean, even just as a default, it seems like many / most plugins would have a single icon to represent themselves throughout the entire experience.

Reply:
I added an option to provide an `icon` property as part of plugin's registration with 15a51cc. It should give it more flexibility because I left the capability to override this icon for the individual components.